### DISCUSSION TITLE
How to show only non-field errors in FormErrors?

### DISCUSSION THREAD

**Discussion by nathan-osman** [vueform/vueform#362] [Questions]
I've got a form that uses `:endpoint` to send a request to the server:

    try {
      await sendApiRequest(form$.requestData)
    } catch (e: any) {
      const data = e.response?.data
      if (data !== undefined) {
        Object.keys(data).forEach((elName) => {
          if (Array.isArray(data[elName]) && form$.el$(elName)) {
            data[elName].forEach((msg) => {
              form$.el$(elName).messageBag.append(msg)
            })
          }
        })
        if ('non_field_errors' in data) {
          form$.messageBag.append(data['non_field_errors'])
        }
      } else {
        // TODO: handle error
      }
    }

This works as expected with one caveat - the field errors are shown both under the field and at the top of the form:

<img src="https://github.com/user-attachments/assets/2bdede85-5e70-45e3-bb18-f35bd44716bb" width="309">

I would just set `displayErrors` to `false` but I do want to be able to add non-field errors. Is there a way to make FormErrors _only_ show non-field errors (ones added with `form$.messageBag.append()`)?

**Comment by adamberecz** [comment]
You can write a plugin for this, targeting `FormErrors` and rewriting the logic that gathers the errors:
https://stackblitz.com/edit/github-jjevkv?file=src%2FApp.vue,vueform.config.js
